src/action.js

```diff
@@ -11,13 +11,26 @@ export async function run() {
   *** ACTION RUN - START ***
   `)
   const MAX_MESSAGE_LENGTH = 280
+  const MAX_MEDIA_ELEMENT = 4
```

Comment:
do we really need to impose a limit on it? where does the limit come from?

Reply:
The limit comes from the X API which states that you can attach up to 4 photos to a Tweet. [X Docs - Upload media](https://developer.x.com/en/docs/x-api/v1/media/upload-media/uploading-media/media-best-practices) 

Also the library used to interact with the `twitter-api-v2` API accepts maximum 4 media_ids in the tweet method
[node-twitter-api-v2](https://github.com/PLhery/node-twitter-api-v2/blob/1e4163eda210573dd49b0dcdcb68cd56a374a2b4/src/types/v2/tweet.definition.v2.ts#L186)

Reply:
ok but why don't we just delegate this limit to it rather than proxying it ourselves?